core/src/main/java/com/oath/oak/Handle.java

```diff
@@ -69,15 +69,13 @@ void put(V newVal, OakSerializer<V> serializer, MemoryManager memoryManager) {
             return;
         }
         int capacity = serializer.calculateSize(newVal);
-        if (this.value.capacity() < capacity) { // can not reuse the existing space
+        if (this.value.remaining() < capacity) { // can not reuse the existing space
```

Comment:
GitHub won't let me comment on unmodified code, so abusing this line.
In `increaseValueCapacity`, line 34 shouldn't it be `value.remaining()` instead of `value.capacity()`?